src/org/gk/util/BrowserLauncher.java

```diff
@@ -96,15 +96,7 @@ public static String getBrowser() {
 	}
 	
 	private static File getPropertyFile() {
-	    File file = null;
-	    try {
-	        file = GKApplicationUtilities.getPropertyFile(PROP_FILE);
-	    }
-	    catch(IOException e) {
-	        System.err.println("BrowserLauncher.getPropertyFile(): " + e);
-	        e.printStackTrace();
-	    }
-	    return file;
+		return GKApplicationUtilities.getPropertyFile(PROP_FILE);
```

Comment:
Was the error handling redundant? Is that why this change was made?

Reply:
Yes, the method called does not throw the exception being caught so I changed its method signature to remove the `throws` portion and removed the error handling here.

Reply:
Ok.